apps/rush-lib/src/api/RushConfiguration.ts

```diff
@@ -33,8 +34,13 @@ import { ExperimentsConfiguration } from './ExperimentsConfiguration';
 import { PackageNameParsers } from './PackageNameParsers';
 import { RepoStateFile } from '../logic/RepoStateFile';
 import { LookupByPath } from '../logic/LookupByPath';
-import { DependencyType, PackageJsonDependency } from './PackageJsonEditor';
 import { RushPluginsConfiguration } from './RushPluginsConfiguration';
+import type * as DependencyAnalyzerModuleType from '../logic/DependencyAnalyzer';
+
+const DependencyAnalyzerModule: typeof DependencyAnalyzerModuleType = Import.lazy(
+  '../logic/DependencyAnalyzer',
+  require
```

Comment:
If we ever want to bundle Rush, we need a better pattern

Reply:
Yeah we will. This may be less of a problem in a Rush bundle, though.